src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ConfigDebugCommandTest.php

```diff
@@ -122,41 +122,36 @@ public function testComplete(array $input, array $expectedSuggestions)
 
         $tester = new CommandCompletionTester($this->application->get('debug:config'));
 
-        $suggestions = $tester->complete($input, 2);
+        $suggestions = $tester->complete($input);
 
-        $this->assertSame($expectedSuggestions, $suggestions);
+        foreach ($expectedSuggestions as $expectedSuggestion) {
+            $this->assertContains($expectedSuggestion, $suggestions);
+        }
     }
 
     public function provideCompletionSuggestions(): \Generator
     {
         yield 'name' => [
             [''],
             [
-                'DefaultConfigTestBundle',
                 'default_config_test',
-                'ExtensionWithoutConfigTestBundle',
                 'extension_without_config_test',
-                'FrameworkBundle',
                 'framework',
-                'TestBundle',
                 'test',
             ],
         ];
 
```

Comment:
You can add the test for BundleName.

```suggestion

        yield 'name (started CamelCase)' => [['Fra'], ['DefaultConfigTestBundle', 'ExtensionWithoutConfigTestBundle', 'FrameworkBundle', 'TestBundle']];
```

Reply:
I added your test suggestion @GromNaN.